org.pgadmin.pgadmin4.yml

```diff
@@ -0,0 +1,150 @@
+app-id: org.pgadmin.pgadmin4
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+rename-desktop-file: pgadmin4.desktop
+rename-icon: pgadmin4
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=session-bus
+  - --socket=system-bus
```

Comment:
needs exception https://docs.flathub.org/docs/for-app-authors/linter#finish-args-arbitrary-dbus-access

Reply:
what's the reason for arbitrary d-bus?

Reply:
I will go back and check, why I've added this, has been a few months :sweat:

Reply:
Logs print this when running, probably why I added this. Not sure if anything is broken without it, I guess we would be fine, to start without having it set and see if somebody finds out what it is.

```
[3:1030/003536.519330:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:1030/003536.519358:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
```

Reply:
No those are harmless.